GeorgiaTech/Api/Controllers/VolumeController.cs

```diff
@@ -0,0 +1,168 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Server;
+using Api.Models;
+using Server.Controllers;
+// For more information on enabling Web API for empty projects, visit https://go.microsoft.com/fwlink/?LinkID=397860
+
+namespace Api.Controllers
+{
+    [Route("[controller]")]
+    public class VolumeController : Controller
+    {
+        private readonly Server.Controllers.VolumeController volumeController;
```

Comment:
Fairly certain this should be returned through the ControllerFactory, though i can be wrong here.